sematic/scheduling/job_scheduler.py

```diff
@@ -215,7 +216,7 @@ def _assert_is_scheduleable(
         ):
             raise StateNotSchedulable(
                 f"The run {run.id} already had an active job "
-                f"{job.namespace}/{job.name} and thus could not be scheduled."
+                f"{job.namespace}/{job.name} and thus could not be scheduled.",
```

Comment:
Does the linter now demand commas after multi-line strings?

Reply:
no, it is indifferent. This is due to another change I made and then reverted (I had added a parameter). I've removed the commas now.